src/backend/api/Fusion.Resources.Domain/Commands/Personnel/RefreshPersonnel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Fusion.Resources.Database.Entities;
+using MediatR;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Fusion.Resources.Domain.Commands
+{
+    public class RefreshPersonnel : TrackableRequest<QueryExternalPersonnelPerson>
+    {
+        private readonly PersonnelId personellId;
+
+        public RefreshPersonnel(PersonnelId personellId)
+        {
+            this.personellId = personellId;
+        }
+
+        public class Handler : IRequestHandler<RefreshPersonnel, QueryExternalPersonnelPerson>
+        {
+            private readonly IProfileService profileService;
+
+            public Handler(IProfileService profileService)
+            {
+                this.profileService = profileService;
+            }
+
+            public async Task<QueryExternalPersonnelPerson> Handle(RefreshPersonnel request, CancellationToken cancellationToken)
+            {
+                var profile = await profileService.RefreshExternalPersonnelAsync(request.personellId.OriginalIdentifier);
+                profile.Disciplines = new List<DbPersonnelDiscipline>();
+
+                return new QueryExternalPersonnelPerson(profile);
```

Comment:
Not quite sure what is going in here? 
Either the command should not return any data, forcing the consumer to run the query, or the command should return proper data? This clears the discipline collection... 

I would rather not return anything, and make the one controller that returns data execute the get query, or i would run a get query at the end of the command. But that would produce a lot of queries when refreshing all, so maybe opt for option 1...

Reply:
Will run a new query to avoid problem.